### PR TITLE
Add target platforms content

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add target platforms content

## QA

- Run ./run to get started
- Go to http://127.0.0.1:8015/target-platforms
- Make sure the copy is the same as the target platforms section in the copy doc

Doc: https://docs.google.com/document/d/1bvQDL9UlLMx8DFNQTCELPB4I3qBNwSV-OAig7z5-Jts/edit#

### CODE REVIEW

File: templates/pages/target-platforms/index.md [hunk 1]
```diff
@@ -7,4 +7,46 @@ page_type: modular
...
+
+Ubuntu is at the heart of many smart devices – from edge computing and wearables to autonomous vehicles and drones.
+
+For fast iterations on your software and hardware stack, we recommend you install Ubuntu Server on your board.
+
+[See available images](https://www.ubuntu.com/download/server)
+
+When you are ready to deploy more advanced prototypes or a production environment, running  your snaps on Ubuntu Core is the best option. Snaps are confined for stricter security and provide automated software updates so your app is always fresh and never broken.
+
+[Get started with Ubuntu Core](developer.ubuntu.com/ubuntu-core)
```
Comment (WillMoggridge) [inline templates/pages/target-platforms/index.md]: This link doesn't actually work. It misses the https, or it needs to be `/core`

File: templates/pages/target-platforms/index.md [hunk 2]
```diff
@@ -7,4 +7,46 @@ page_type: modular
...
+
+[Download Ubuntu Desktop](https://www.ubuntu.com/download/desktop)
+
+For testers and early adopters, the daily desktop build version can be downloaded anytime.
+
+[Get the Ubuntu daily build](http://cdimage.ubuntu.com/daily-live/current/)
+
+With Ubuntu Make, you can easily install the latest stable version of the most popular IDEs, regardless of your language and technology of choice.
+
+[Install and use Ubuntu Make](#)
```
Comment (WillMoggridge) [inline templates/pages/target-platforms/index.md:26]: Do we have a link for this? Otherwise I'm worried we may accidentally leave it without one.

**Review by WillMoggridge — CHANGES_REQUESTED** [review]

File: templates/pages/target-platforms/index.md (hunk 2 above)
Comment (grahambancroft) [reply templates/pages/target-platforms/index.md:26]: not yet, I did add a comment to the doc though requesting one

File: templates/pages/target-platforms/index.md (hunk 1 above)
Comment (grahambancroft) [reply templates/pages/target-platforms/index.md]: Fixed

**Review by WillMoggridge — APPROVED** [review]

**Pull request merged**